### PR TITLE
Make setDir shouldcallparent

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
cihgnels

image

### CODE REVIEW

File: code/game/objects/items/cardboard_cutouts.dm [hunk 1]
```diff
@@ -181,7 +181,8 @@
 	return 1
 
 /obj/item/cardboard_cutout/setDir(newdir)
-	dir = SOUTH
+	newdir = SOUTH
+	return ..()
```
Comment (Dennok) [inline code/game/objects/items/cardboard_cutouts.dm]: This intended?

Comment (Dennok) [inline code/game/objects/items/cardboard_cutouts.dm]: I think you need `return ..(SOUTH)` here.
newdir only var in this proc and it dont put it nowhere.
or place `return ..(newdir)`

Comment (spookydonut) [reply code/game/objects/items/cardboard_cutouts.dm]: it works fine as is.

Comment (Dennok) [reply code/game/objects/items/cardboard_cutouts.dm]: Really

Comment (MrPerson) [reply code/game/objects/items/cardboard_cutouts.dm]: ..() automatically forwards arguments

Comment (XDTM) [reply code/game/objects/items/cardboard_cutouts.dm]: I think the point is that this proc is overriding the argument by always setting SOUTH, but parent calls will inherit the now-wrong argument instead. I don't know if it has any potential to cause issues, but it should indeed be ..(SOUTH) in my opinion

**Review by ninjanomnom — APPROVED** [review]

**Pull request merged**